alaoweb/2020.alaoweb.org
# Update poster data structure

* closes #171
* replaces video param with more comprehensive options `session-contents` and `supplemental-docs`
* adds commented-out template for those params in each poster
* updates page layout to accommodate/display data with this structure 

### To Test:
check out: https://deploy-preview-175--alao-production.netlify.app/posters/libcal-to-circulate OR http://localhost:4000/posters/libcal-to-circulate (note: this video is not the real video or supplemental-docs for the presentation, though the image and text might be.)
